snapshot-labs/sx-monorepo
# fix: fix empty voting period being overriden

### Summary



Closes: #1012 
Toward https://github.com/snapshot-labs/workflow/issues/282

This PR fix an issue where the voting period is saved as empty for offchain spaces, but shows as 7 days on the UI.

Issue was fixed by allowing `0` as voting period value (which means no default enforcing, and  up to the proposer), with a fallback to 3 days (like on v1)

### How to test

1. Go to an offchain space settings
2. Save the voting period as 0
3. It saves as 0 (previously, will always show 7 instead of 0)
4. Go to the Editor
5. Proposal end time should be set to 3 days after start
